File: problems/problems_2563/Solution.java
package problems.problems_2563;

import com.alibaba.fastjson.JSON;
import java.util.*;
import qubhjava.BaseSolution;


public class Solution extends BaseSolution {
    public long countFairPairs(int[] nums, int lower, int upper) {
        
    }

    @Override
    public Object solve(String[] inputJsonValues) {
        int[] nums = jsonArrayToIntArray(inputJsonValues[0]);
		int lower = Integer.parseInt(inputJsonValues[1]);
		int upper = Integer.parseInt(inputJsonValues[2]);
        return JSON.toJSON(countFairPairs(nums, lower, upper));
    }
}

File: problems/problems_2563/solution.go
package problem2563

import (
	"encoding/json"
	"log"
	"strings"
)

func countFairPairs(nums []int, lower int, upper int) int64 {
    
}

func Solve(inputJsonValues string) interface{} {
	inputValues := strings.Split(inputJsonValues, "\n")
	var nums []int
	var lower int
	var upper int

	if err := json.Unmarshal([]byte(inputValues[0]), &nums); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[1]), &lower); err != nil {
		log.Fatal(err)
	}
	if err := json.Unmarshal([]byte(inputValues[2]), &upper); err != nil {
		log.Fatal(err)
	}

	return countFairPairs(nums, lower, upper)
}
